src/contrast/pages/UploadNewPaper/ExperimentsSection/Findings.jsx

```diff
@@ -108,14 +106,15 @@ export default function Findings({
       );
     }
     if (field?.family == families["Frequency"]) {
-      return [
-        field.direction,
-        field.onset,
-        field.offset,
-        field.band_higher_bound,
-        field.band_lower_bound,
-        field.analysis_type,
-      ].every((condition) => Boolean(condition) === true);
+      return (
+        [
+          field.direction,
+          field.band_higher_bound,
+          field.band_lower_bound,
+          field.analysis_type,
+        ].every((condition) => Boolean(condition) === true) &&
+        Boolean(field.onset) === Boolean(field.offset)
```

Comment:
again the condition seems wrong, what you probably want to check is that while they don't have to exist (unlike the 4 others here) if one exists, you also need to second one to exist